packages/core/src/layout/InfoCard/InfoCard.tsx

```diff
@@ -0,0 +1,228 @@
+      flexDirection: 'column',
+    },
+    widget: {
+      height: 430,
+    },
+    fullHeight: {
+      height: '100%',
+    },
+    height100: {
+      display: 'flex',
+      flexDirection: 'column',
+      height: 'calc(100% - 10px)', // for pages without content header
+      marginBottom: '10px',
+    },
+    contentheader: {
+      height: 'calc(100% - 40px)', // for pages with content header
+    },
+    contentheadertabs: {
+      height: 'calc(100% - 97px)', // for pages with content header and tabs (Tingle)
+    },
+    noShrink: {
+      flexShrink: 0,
+    },
+    minheight300: {
+      minHeight: 300,
+      overflow: 'initial',
+    },
+  },
+  cardContent: {
+    widget: {
+      overflowY: 'auto',
+      height: 332,
+      width: '100%',
+    },
+    fullHeight: {
+      height: 'calc(100% - 50px)',
+    },
+    height100: {
+      height: 'calc(100% - 50px)',
+    },
+    contentRow: {
+      display: 'flex',
+      flexDirection: 'row',
+    },
+  },
+};
+
+/**
+ * InfoCard is used to display a paper-styled block on the screen, similar to a panel.
+ *
+ * You can custom style an InfoCard with the 'style' (outer container) and 'cardStyle' (inner container)
+ * styles.
+ *
+ * The InfoCard serves as an error boundary. As a result, if you provide a 'slackChannel' property this
+ * specifies the channel to display in the error component that is displayed if an error occurs
+ * in any descendent components.
+ *
+ * By default the InfoCard has no custom layout of its children, but is treated as a block element. A
+ * couple common variants are provided and can be specified via the variant property:
+ *
+ * Display the card full height suitable for DataGrid:
+ *
+ *   <InfoCard variant="height100">...</InfoCard>
+ *
+ * Variants can be combined in a whitespace delimited list like so:
+ *
+ *   <InfoCard variant="noShrink">...</InfoCard>
+ */
+type Props = {
+  title?: ReactNode;
+  subheader?: ReactNode;
+  divider?: boolean;
+  deepLink?: object;
+  slackChannel?: string;
+  variant?: string;
+  style?: object;
+  cardStyle?: object;
+  children: ReactNode;
+  headerStyle?: object;
+  headerProps?: object;
+  actionsClassName?: string;
+  actions?: ReactNode;
+  cardClassName?: string;
+  actionsTopRight?: ReactNode;
+};
+
+const InfoCard: FC<Props> = ({
+  title,
+  subheader,
+  divider,
+  deepLink,
+  slackChannel = '#backstage',
+  variant,
+  style,
+  cardStyle,
+  children,
+  headerStyle,
+  headerProps,
+  actionsClassName,
+  actions,
+  cardClassName,
+  actionsTopRight,
+}) => {
+  const classes = useStyles();
+  if (style) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      'InfoCard: using `style` property directly, consider migrating your style to variant in InfoCard',
+    );
+  }
+  if (cardStyle) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      'InfoCard: using `cardStyle` property directly, consider migrating your style to variant in InfoCard',
```

Comment:
I think we should just get rid of the style and cardStyle props tbh. That or removing the warning. There's no existing code to migrate so it doesn't make much sense :grin:

Reply:
Sounds good. It has a looot of props so any pruning of props is good imo. :)